aspnetcore/blazor/debug.md

```diff
@@ -28,11 +28,23 @@ Available scenarios include:
 
 For now, you *can't*:
 
+::: moniker range=">= aspnetcore-6.0"
+
+* Break on unhandled exceptions.
+* Hit breakpoints during app startup before the debug proxy is running. This includes breakpoints in `Program.Main` (`Program.cs`) and breakpoints in the [`OnInitialized{Async}` lifecycle methods](xref:blazor/components/lifecycle#component-initialization-oninitializedasync) of components that are loaded by the first page requested from the app.
+* Automatically rebuild the backend `*Server*` app of a hosted Blazor WebAssembly solution during debugging, for example by running the app with [`dotnet watch run`](xref:tutorials/dotnet-watch).
+
+::: moniker-end
+
+::: moniker range="< aspnetcore-6.0"
+
 * Break on unhandled exceptions.
 * Hit breakpoints during app startup before the debug proxy is running. This includes breakpoints in `Program.Main` (`Program.cs`) and breakpoints in the [`OnInitialized{Async}` lifecycle methods](xref:blazor/components/lifecycle#component-initialization-oninitializedasync) of components that are loaded by the first page requested from the app.
 * Debug in non-local scenarios (for example, [Windows Subsystem for Linux (WSL)](/windows/wsl/) or [Visual Studio Codespaces](/visualstudio/codespaces/overview/what-is-vsonline)).
```

Comment:
We don't support debugging remotely in Visual Studio yet.

Also, there's a bug that prevents the WSL/VS Code scenario from working fully so I would omit this for now.

Reply:
😕 ... That's a bit confusing. I can't grok what "yet" and "for now" mean WRT today versus at GA. We can leave it if at GA things will work and this content will be accurate at that time.

The lead-in for this list is **_For now, you *can't*:_**. I assumed from the PU PR that as of 6.0, one **_can_** ...

> Debug in non-local scenarios (for example, \[Windows Subsystem for Linux (WSL)](/windows/wsl/) or \[Visual Studio Codespaces](/visualstudio/codespaces/overview/what-is-vsonline)).

Therefore, this line will only appear in content for <6.0.

If I still fail to grasp what's changing, you'll have to explain it out a bit more.

Reply:
Yeah, sorry for the confusion.

>  I assumed from the PU PR that as of 6.0, one can .

Nope this is not true. We can basically close this PR.

Context: we pushed the change to support remote debugging in .NET 6 Preview 1. It worked for a while then a downstream component changes their API in .NET 6 Preview 3 and it has be broken since then. There are no plans to fix for .NET 6 at the moment.

Reply:
Ah ... gotcha ... and I'll remove this from the 6.0 tracking issue as well.